packages/contracts-bedrock/test/mocks/TestAccount.sol

```diff
@@ -19,12 +19,13 @@ contract TestAccount {
     {
         (version, txHash, transaction);
         // limited to 48 bits
-        uint64 validUntil = type(uint64).max & 0xFFFFFFFFFFFF;
-        uint64 validAfter = 0;
-        validationData =
-            (bytes32(MAGIC_VALUE_SENDER) |
-            bytes32(uint256(validUntil) << (6 * 8)) |
-            bytes32(uint256(validAfter)));
+        uint48 validUntil = 0;
```

Comment:
I was wondering why you set the until value to 0?

Reply:
If both values of `validUntil` and `validAfter` are 0, it will be recognized as `[0, type(uint48).max]` automatically at `validateValidityTimeRange` function in geth.

Reply:
Oh, right.
Passing validUntil = 0 and validAfter = 0 in the specification disables the check. So this part applies